kem/sike/sikep434/sike.go

```diff
@@ -1,6 +1,13 @@
 // Code generated from pkg.templ.go. DO NOT EDIT.
 
-// Package sikep434 implements the key encapsulation mechanism SIKEp434.
+// Package sikep434 is deprecated, it implements the key encapsulation mechanism SIKEp434.
```

Comment:
```suggestion
// Package sikep434 is deprecated. It implements the key encapsulation mechanism SIKEp434.
```

Reply:
period cannot be used until the end of sentence because the parser for documentation does not process this properly.